extensions/s3-extensions/src/main/java/io/druid/storage/s3/S3DataSegmentPusher.java

```diff
@@ -66,8 +68,10 @@ public String getPathForHadoop(String dataSource)
   @Override
   public DataSegment push(final File indexFilesDir, final DataSegment inSegment) throws IOException
   {
-    log.info("Uploading [%s] to S3", indexFilesDir);
     final String s3Path = S3Utils.constructSegmentPath(config.getBaseKey(), inSegment);
+
+    log.info("Uploading segment[%s] to S3 at path[%s]", inSegment.getIdentifier(), s3Path);
```

Comment:
minor nitpick again, can we make those log messages consistent between deep storage implementations?